packages/front-end/components/Settings/AdminSetPasswordModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Field from "../Forms/Field";
+import Modal from "../Modal";
+import { useForm } from "react-hook-form";
+import { MemberRole } from "back-end/types/organization";
+import { useAuth } from "../../services/auth";
+
+type Member = { id: string; name: string; email: string; role: MemberRole };
```

Comment:
We have this exact same type declared in MemberList.  We should just have a single type declaration that we share instead.  Also, we already have a type called `Member` in `organization.d.ts` which just has id and role, so maybe use a different type name here.

Reply:
@jdorn Good catch. I've updated this from `Member` to `MemberInfo` and it's currently defined within `MemberList.tsx` but I'm wondering if I should define it in `organization.d.ts` and then import it in `MemberList` and `AdminSetPasswordModal`. Let me know if you have a strong opinion either way.